src/Resolver.js

```diff
@@ -185,4 +185,8 @@ export default class Resolver {
       return React.renderToStaticMarkup(context);
     });
   }
+
+  static decorate(props = {}) {
```

Comment:
I should probably rename this to `options` (which is what I started doing in another branch of this).

The fact that `resolve` gets passed in as a prop in the container doesn't matter.